spec/integration_spec.rb

```diff
@@ -349,6 +359,13 @@ def public?
   end
 end
 
+describe 'DisabledIndexing' do
```

Comment:
A spec to add:
- other classes should call get_settings

A spec to maybe add?:
- DisabledIndexing should check_settings

Nit: the case of `DisabledIndexing` is strange.  I would have thought `DisabledIndex` maybe?  There are other classes that are pluralized in this file and some that are singular.  It's unclear to me if `Indexing` is the correct choice or something else is preferable.

Reply:
> A spec to add:
> * other classes should call get_settings

This is tested implicitly, but I've added an explicit test in d77de87ee2080d2c8122453e289530f5410cb7a6.

Regarding the naming, I went with `Indexing` to refer to the action of indexing, and preventing confusion with `Index` objects. It follows the same convention as other `disabled_*` 'models' we have in these tests 🙂